### ISSUE TITLE
Reviews/stats stopped downloading, app crashes when I report

### ISSUE THREAD

**Issue by alexblack** [AndlyticsProject/andlytics#697]
Just started late last night

![screenshot_2015-08-12-06-57-01](https://cloud.githubusercontent.com/assets/148930/9225665/8891eae6-40bf-11e5-8bd5-86710bbc670c.png)

**Comment by daparker** [comment]
I'm seeing the same thing.  When I ran Andlytics with Logcat, I found a few errors are showing up.  When the app first launches, I see this:

```
W/ACRA(18899): Error : 
W/ACRA(18899): org.acra.ACRAConfigurationException: TOAST mode: you have to define the resToastText parameter in your application @ReportsCrashes() annotation.
W/ACRA(18899): 	at org.acra.ACRA.checkCrashResources(ACRA.java:259)
W/ACRA(18899): 	at org.acra.ACRA.init(ACRA.java:171)
W/ACRA(18899): 	at org.acra.ACRA.init(ACRA.java:135)
W/ACRA(18899): 	at org.acra.ACRA.init(ACRA.java:120)
W/ACRA(18899): 	at com.github.andlyticsproject.AndlyticsApp.initAcra(AndlyticsApp.java:48)
W/ACRA(18899): 	at com.github.andlyticsproject.AndlyticsApp.onCreate(AndlyticsApp.java:31)
W/ACRA(18899): 	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1020)
W/ACRA(18899): 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5113)
W/ACRA(18899): 	at android.app.ActivityThread.access$1600(ActivityThread.java:177)
W/ACRA(18899): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1509)
W/ACRA(18899): 	at android.os.Handler.dispatchMessage(Handler.java:102)
W/ACRA(18899): 	at android.os.Looper.loop(Looper.java:145)
W/ACRA(18899): 	at android.app.ActivityThread.main(ActivityThread.java:5942)
W/ACRA(18899): 	at java.lang.reflect.Method.invoke(Native Method)
W/ACRA(18899): 	at java.lang.reflect.Method.invoke(Method.java:372)
W/ACRA(18899): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1399)
W/ACRA(18899): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1194)
W/AndlyticsApp(18899): ACRA.init() called more than once?: Cannot access ErrorReporter before ACRA#init
W/AndlyticsApp(18899): java.lang.IllegalStateException: Cannot access ErrorReporter before ACRA#init
W/AndlyticsApp(18899): 	at org.acra.ACRA.getErrorReporter(ACRA.java:221)
W/AndlyticsApp(18899): 	at com.github.andlyticsproject.AndlyticsApp.initAcra(AndlyticsApp.java:52)
W/AndlyticsApp(18899): 	at com.github.andlyticsproject.AndlyticsApp.onCreate(AndlyticsApp.java:31)
W/AndlyticsApp(18899): 	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1020)
W/AndlyticsApp(18899): 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5113)
W/AndlyticsApp(18899): 	at android.app.ActivityThread.access$1600(ActivityThread.java:177)
W/AndlyticsApp(18899): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1509)
W/AndlyticsApp(18899): 	at android.os.Handler.dispatchMessage(Handler.java:102)
W/AndlyticsApp(18899): 	at android.os.Looper.loop(Looper.java:145)
W/AndlyticsApp(18899): 	at android.app.ActivityThread.main(ActivityThread.java:5942)
W/AndlyticsApp(18899): 	at java.lang.reflect.Method.invoke(Native Method)
W/AndlyticsApp(18899): 	at java.lang.reflect.Method.invoke(Method.java:372)
W/AndlyticsApp(18899): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1399)
W/AndlyticsApp(18899): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1194)
```

Then there's the JSON failure:

```
E/Main(18899): Error while requesting developer console : com.github.andlyticsproject.console.v2.DevConsoleV2Protocol.parseRatingsResponse(DevConsoleV2Protocol.java:212)
E/Main(18899): com.github.andlyticsproject.console.v2.DevConsoleV2.fetchRatings(DevConsoleV2.java:290)
E/Main(18899): com.github.andlyticsproject.console.v2.DevConsoleV2.fetchAppInfosAndStatistics(DevConsoleV2.java:122)
E/Main(18899): com.github.andlyticsproject.console.v2.DevConsoleV2.getAppInfo(DevConsoleV2.java:103)
E/Main(18899): com.github.andlyticsproject.Main$LoadRemoteEntries.doInBackground(Main.java:521)
E/Main(18899): com.github.andlyticsproject.Main$LoadRemoteEntries.doInBackground(Main.java:488)
E/Main(18899): android.os.AsyncTask$2.call(AsyncTask.java:288)
E/Main(18899): java.util.concurrent.FutureTask.run(FutureTask.java:237)
E/Main(18899): java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
E/Main(18899): java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
E/Main(18899): java.lang.Thread.run(Thread.java:818)
E/Main(18899): org.json.JSONObject.get(JSONObject.java:389)
E/Main(18899): org.json.JSONObject.getInt(JSONObject.java:478)
E/Main(18899): com.github.andlyticsproject.console.v2.JsonParser.parseRatings(JsonParser.java:63)
E/Main(18899): com.github.andlyticsproject.console.v2.DevConsoleV2Protocol.parseRatingsResponse(DevConsoleV2Protocol.java:209)
E/Main(18899): com.github.andlyticsproject.console.v2.DevConsoleV2.fetchRatings(DevConsoleV2.java:290)
E/Main(18899): com.github.andlyticsproject.console.v2.DevConsoleV2.fetchAppInfosAndStatistics(DevConsoleV2.java:122)
E/Main(18899): com.github.andlyticsproject.console.v2.DevConsoleV2.getAppInfo(DevConsoleV2.java:103)
E/Main(18899): com.github.andlyticsproject.Main$LoadRemoteEntries.doInBackground(Main.java:521)
E/Main(18899): com.github.andlyticsproject.Main$LoadRemoteEntries.doInBackground(Main.java:488)
E/Main(18899): android.os.AsyncTask$2.call(AsyncTask.java:288)
E/Main(18899): java.util.concurrent.FutureTask.run(FutureTask.java:237)
E/Main(18899): java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
E/Main(18899): java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
E/Main(18899): java.lang.Thread.run(Thread.java:818)
E/Main(18899): Error while requesting developer console : {"result":{"1":[{"1":"com.dparker.apps.checkvalve","7":"54","8":{"1":"2","2":"2","3":"6","4":"14","5":"73"}}]},"xsrf":"AMtNNDEUQl6mAsYk8Gppt_QZTDrt9vldnA:1439388793348"}
E/Main(18899): com.github.andlyticsproject.console.DevConsoleProtocolException: {"result":{"1":[{"1":"com.dparker.apps.checkvalve","7":"54","8":{"1":"2","2":"2","3":"6","4":"14","5":"73"}}]},"xsrf":"AMtNNDEUQl6mAsYk8Gppt_QZTDrt9vldnA:1439388793348"}
E/Main(18899): 	at com.github.andlyticsproject.console.v2.DevConsoleV2Protocol.parseRatingsResponse(DevConsoleV2Protocol.java:212)
E/Main(18899): 	at com.github.andlyticsproject.console.v2.DevConsoleV2.fetchRatings(DevConsoleV2.java:290)
E/Main(18899): 	at com.github.andlyticsproject.console.v2.DevConsoleV2.fetchAppInfosAndStatistics(DevConsoleV2.java:122)
E/Main(18899): 	at com.github.andlyticsproject.console.v2.DevConsoleV2.getAppInfo(DevConsoleV2.java:103)
E/Main(18899): 	at com.github.andlyticsproject.Main$LoadRemoteEntries.doInBackground(Main.java:521)
E/Main(18899): 	at com.github.andlyticsproject.Main$LoadRemoteEntries.doInBackground(Main.java:488)
E/Main(18899): 	at android.os.AsyncTask$2.call(AsyncTask.java:288)
E/Main(18899): 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
E/Main(18899): 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
E/Main(18899): 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
E/Main(18899): 	at java.lang.Thread.run(Thread.java:818)
E/Main(18899): Caused by: org.json.JSONException: No value for 2
E/Main(18899): 	at org.json.JSONObject.get(JSONObject.java:389)
E/Main(18899): 	at org.json.JSONObject.getInt(JSONObject.java:478)
E/Main(18899): 	at com.github.andlyticsproject.console.v2.JsonParser.parseRatings(JsonParser.java:63)
E/Main(18899): 	at com.github.andlyticsproject.console.v2.DevConsoleV2Protocol.parseRatingsResponse(DevConsoleV2Protocol.java:209)
E/Main(18899): 	... 10 more
```

And then there's this when you try to send the error report:

```
E/AndroidRuntime(18899): FATAL EXCEPTION: main
E/AndroidRuntime(18899): Process: com.github.andlyticsproject, PID: 18899
E/AndroidRuntime(18899): java.lang.IllegalStateException: Cannot access ErrorReporter before ACRA#init
E/AndroidRuntime(18899): 	at org.acra.ACRA.getErrorReporter(ACRA.java:221)
E/AndroidRuntime(18899): 	at com.github.andlyticsproject.BaseActivity.sendAracReport(BaseActivity.java:232)
E/AndroidRuntime(18899): 	at com.github.andlyticsproject.BaseActivity.access$000(BaseActivity.java:35)
E/AndroidRuntime(18899): 	at com.github.andlyticsproject.BaseActivity$5$1.run(BaseActivity.java:251)
E/AndroidRuntime(18899): 	at java.lang.Thread.run(Thread.java:818)
E/AndroidRuntime(18899): 	at com.github.andlyticsproject.BaseActivity$5.onClick(BaseActivity.java:254)
E/AndroidRuntime(18899): 	at com.github.andlyticsproject.dialog.CrashDialog$CrashDialogBuilder$1.onClick(CrashDialog.java:168)
E/AndroidRuntime(18899): 	at android.view.View.performClick(View.java:5197)
E/AndroidRuntime(18899): 	at android.view.View$PerformClick.run(View.java:20926)
E/AndroidRuntime(18899): 	at android.os.Handler.handleCallback(Handler.java:739)
E/AndroidRuntime(18899): 	at android.os.Handler.dispatchMessage(Handler.java:95)
E/AndroidRuntime(18899): 	at android.os.Looper.loop(Looper.java:145)
E/AndroidRuntime(18899): 	at android.app.ActivityThread.main(ActivityThread.java:5942)
E/AndroidRuntime(18899): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(18899): 	at java.lang.reflect.Method.invoke(Method.java:372)
E/AndroidRuntime(18899): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1399)
E/AndroidRuntime(18899): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1194)
```

I can post the entire logcat session if needed.

**Comment by TheNephilim88** [comment]
JSON failure fix is here: https://github.com/AndlyticsProject/andlytics/pull/696

**Comment by willlunniss** [comment]
The parsing issue and the acra issue have now been fixed and pushed out to Google Play as 2.7.3, thank you to @TheNephilim88 for providing the parsing fix.
